### DISCUSSION TITLE
Missing flags for multiple countries?

### DISCUSSION THREAD

**Discussion by will-darwin** [axelboc/anki-ultimate-geography#610] [General]
Hi,

Seems like there may be some missing flags from this collection? I would be happy to assist in fixing this but wanted to check as there may be a valid reason. As far as I can tell, the following flags are missing:

- Bermuda
- Guadeloupe
- Zanzibar
- Bali
- British Virgin Islands
- Turks and Caicos Islands
- Cayman Islands
- Anguilla
- Azores
- French Guiana
- Gibraltar
- Guernsey
- Isle of Man
- Jersey
- Madeira
- American Samoa
- Northern Mariana Islands
- Sint Maarten
- Sardinia
- Kaliningrad Oblast
- Martinique
- Java
- Corsica
- Saint Martin
- Wallis and Futuna
- Akrotiri and Dhekelia
- Svalbard

I'm sure some of the flags I can see when I google these countries may not be official but is this the case for all of them? I have often used geography quiz apps which show a flag I've never seen in Ultimate Geography, which prompted my question.

**Comment by aplaice** [comment]
Thanks for bringing this up!

The flags and capitals for them are [deliberately excluded](https://github.com/anki-geo/ultimate-geography/blob/master/CONTRIBUTING.md#dependent-territories) (the relevant entities are small dependent states, islands, exclaves or the like).

(The rationale for the small dependent states etc. is that most people won't be too interested in them, so just having the country<->map cards — i.e. knowing approximately where the "entity" is — is sufficient.)

(For some of the larger islands, the idea is that they're of interest for physical geography, rather than political geography reasons, so again the flag/capital is a bit out of scope.)

Some of them [used to be in the deck until they were removed](https://github.com/anki-geo/ultimate-geography/pull/312) and you can find them [here, as the "Legacy" deck](https://github.com/anki-geo/ultimate-geography/releases/tag/v4.0).

<hr/>

They're also being (re?)added to the more extensive (WIP and unreleased) ["hardcore geography" deck](https://github.com/anki-geo/hardcore-geography).

(Edit: If you'd like, you're very welcome to contribute! :))

**Comment by gombost** [comment]
I'm completely baffled by this.Everything should be added and people who are not interested in some of the cards should leave those suspended. I'm grateful for the work that people have been putting into this deck but this is "incomplete" not "ultimate" geography.

**Reply by axelboc** [reply]
Unfortunately, this would not provide a great experience for people who are not power users of Anki. Our goal is to provide a good default for most users, and the number of positive reviews on the [shared deck](https://ankiweb.net/shared/info/2109889812) is testimony that we've made the right choices so far.

As @aplaice mentioned, work is currently happening on a "Hardcore Geography" deck, which you're very welcome to help out with.
